src/utils/validation.ts

```diff
@@ -31,7 +31,7 @@ export const validation = {
   isValidUserPrincipalName(upn: string): boolean {
     const upnRegEx = new RegExp(/^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$/i);
 
-    return upnRegEx.test(upn);
+    return upnRegEx.test(upn) || upn.toLocaleLowerCase().trim() === "@meusername";
```

Comment:
When replacing the access token we are using `toLowerCase` instead of `toLocaleLowerCase`, so I suggest that we use that one (we should update `isValidGuid` as well.